index.html

```diff
@@ -13,7 +13,7 @@ <h1>Welcome to Aun's Page</h1> <!-- change this heading to your name -->
 
     <main>
       <!-- ====== ABOUT SECTION (edit the text) ====== -->
-      <h2>About Me</h2>
+      <h2>About Me</h2> <!-- TODO: improve About text -->
       <p>I am a student at Queen’s School of Computing. I’m interested in Artificial Intelligence. My hobbies are gardening and basketball! I love coding in Java, and I also love making games as a side hobby.</p>
```

Comment:
```suggestion
      <p>I am a student at Queen’s School of Computing. I’m interested in Artificial Intelligence. My hobbies are gardening and basketball! I love coding in Java, and I also love making games as a side hobby.</p>
```Really nice information about yourself